### PR TITLE
Fix build for UcsSDK

### PR DESCRIPTION
**Pull request by tbachman**
Signed-off-by: Thomas Bachman tbachman@yahoo.com

### CODE REVIEW

File: UcsPythonSDK/debian/python-ucssdk.debhelper.log [hunk 1]
```diff
@@ -0,0 +1,17 @@
+dh_auto_configure
```
Comment (amittbose) [inline UcsPythonSDK/debian/python-ucssdk.debhelper.log]: I don't think this file should be checked-in

Comment (tbachman) [inline UcsPythonSDK/debian/python-ucssdk.debhelper.log]: You're right :-). Will fix.

**Comment by tbachman** [comment]
Updated patch, based on reviewer comments.

**Comment by tbachman** [comment]
Updated patch removes unused files, fixes contact/owner, etc.

**Pull request merged**